lib/apparmortest.pm

```diff
@@ -123,7 +123,8 @@ sub aa_status_stdout_check {
 }
 
 sub ip_fetch {
-    my $ip = script_output("hostname -I | cut -d ' ' -f1");
+    # "# hostname -i/-I" can not work in some cases
+    my $ip = script_output("ip -4 address | grep inet | grep -E \'eth0|ens\' | sed 's/^[ \t]*//g' | cut -d ' ' -f2 | cut -d '/' -f1");
```

Comment:
less letters to emulate typying :)
```suggestion
    my $ip = script_output("ip -4 -f inet -o a | grep -E \'eth0|ens\' | sed -n 's/.*inet \([0-9.]\+\).*/\1/p'");
```

Reply:
Awesome, thanks! revised.

Reply:
@asmorodskyi 
I am still debugging this code as it can not work in openQA (can work in shell), please do not merge the PR, thanks, I will continue debugging tomorrow.

Reply:
can you point me to failed run ?

Reply:
I checked the video, the console always shows **# ip -4 -f inet -o a | grep -E 'eth0|ens' | sed -n 's"**,
other characters are omitted, it might be a openQA backend issue  on script_output();

Reply:
FYI: find the issue at the end of the video, http://10.67.19.89/tests/1185/file/video.ogv#t=21.79,21.83

Reply:
Done, and the verification run is PASS: http://10.67.19.89/tests/1192,

Reply:
> I checked the video, the console always shows **# ip -4 -f inet -o a | grep -E 'eth0|ens' | sed -n 's"**,
> other characters are omitted, it might be a openQA backend issue on script_output();

Sorry for the misleading, it is not an openQA backend issue, it is test code issue: some of the "escape characters" the code are not translated correctly.